plugins/ltac/rewrite.ml

```diff
@@ -1973,42 +1973,47 @@ let warn_add_morphism_deprecated =
   CWarnings.create ~name:"add-morphism" ~category:"deprecated" (fun () ->
       Pp.(str "Add Morphism f : id is deprecated, please use Add Morphism f with signature (...) as id"))
 
-let add_morphism_infer atts m n : Proof_global.t option =
+let add_morphism_as_parameter atts m n : unit =
   warn_add_morphism_deprecated ?loc:m.CAst.loc ();
```

Comment:
This will produce an incorrect message since it's not called by `Add Morphism`

Reply:
oops, thanks!

Reply:
removed the warning. as far as I know `Add Morphism` is deprecated but the alternative is not viable. This new command does not have an alternative either, so I've just removed the warning in that case.